scripts/deploy-auction.ts

```diff
@@ -0,0 +1,85 @@
+import { SuiClient, getFullnodeUrl } from "@mysten/sui/client";
+
+import * as dotenv from "dotenv";
+import { Transaction } from "@mysten/sui/transactions";
+import { Ed25519Keypair } from "@mysten/sui/keypairs/ed25519";
+
+
+dotenv.config();
+
+async function main() {
+	const { MNEMONIC, PACKAGE_ID, NETWORK, START_MS, DURATION_MS, AUCTION_SIZE} = process.env;
+
+	if (!MNEMONIC || !PACKAGE_ID || !NETWORK || !START_MS || !DURATION_MS || !AUCTION_SIZE) {
+		console.error("❌ Error: Missing required environment variables. Check your .env file.");
+		process.exit(1);
+	}
+
+	const rpcUrl = getFullnodeUrl(NETWORK as "mainnet" | "testnet" | "devnet" | "localnet");
+	const client = new SuiClient({ url: rpcUrl });
+	const keypair = Ed25519Keypair.deriveKeypair(MNEMONIC);
+
+	console.log(`📦 Package ID: ${PACKAGE_ID}`);
+	if (!NETWORK) {
+		console.error("❌ Error: SUI_RPC_URL is not set in your .env file.");
+		process.exit(1);
+	}
```

Comment:
lines from "main" until here is copy paste. Let's export it to the common function, let's say  in "suiinit.ts". In other files I made example how to import scripts (note: you need to use ".js" suffix in imports even though files are ".ts")